api/envoy/config/filter/http/buffer/v2/buffer.proto

```diff
@@ -19,12 +19,7 @@ message Buffer {
   // manager will stop buffering and return a 413 response.
   google.protobuf.UInt32Value max_request_bytes = 1 [(validate.rules).uint32.gt = 0];
 
-  // The maximum number of seconds that the filter will wait for a complete
-  // request before returning a 408 response.
-  // deprecated in favor of http connection manager of :ref:request timeouts
-  // <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.request_timeout>
-  google.protobuf.Duration max_request_time = 2
-      [deprecated = true, (validate.rules).duration = {gt: {}}, (gogoproto.stdduration) = true];
+  google.protobuf.Duration max_request_time = 2 [deprecated = true];
```

Comment:
Hmm yeah, we should have deprecated this in the previous PR. We did mark this as deprecated in release notes for the last release, right? If so, I don't think it matters. But you should remove this field and reserve it.

Reply:
It was marked as deprecated in the previous PR, I just removed the extra validations. I'll remove the field and reserve it now.